File: C Programs/Problems on Pattern Printing/Rows & Columns/Pattern 2.c
/* Accept number of rows and number of columns from user and display below pattern.

Input : iRow = 4 iCol = 3
Output :
 1 2 3
 1 2 3
 1 2 3
 1 2 3

 */
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i=0;
    int j=0;

    for(i=1;i<=iRow;i++)
    {
        for(j=1;j<=iCol;j++)
        {
            printf("%d\t",j);
        }
        printf("\n");
    }
}

int main()
{
 int iRows = 0;
 int iColumns = 0;

 printf("Enter number of rows: ");
 scanf("%d",&iRows);
 printf("Enter number of columns: ");
 scanf("%d",&iColumns);

 Pattern(iRows,iColumns);

 return 0;
}

File: Java Programs/Problems on Pattern Printing/Programs to print Alphabetical Square pattern/Pattern 2.java
/* Write Java program which accept number of rows and number of columns from user and display below pattern.

Input : iRow = 4 iCol = 4
Output : 
 A B C D
 a b c d
 A B C D
 a b c d 

*/

import java.lang.*;
import java.util.*;

class Pattern
{
    void PrintPattern(int iRow,int iCol)
    {
        
        for (int i=1;i<=iRow;i++)
         {
            char ch='A';
            char cha='a';
            for(int j=1;j<=iCol;j++)
             {
                if(i%2==0)
                 {
                    System.out.print(cha+"\t");
                    cha++;
                 }
                else
                 {
                    System.out.print(ch+"\t");
                    ch++;
                 }
            }
            System.out.println();           
        }
    }
}

class Demo
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);
        Pattern pobj=new Pattern();
        
        System.out.println("Enter no of rows: ");
        int iRow=sobj.nextInt();
        
        System.out.println("Enter no of columns: ");
        int iCol=sobj.nextInt();
        
        pobj.PrintPattern(iRow,iCol);
    }
}
